homeassistant/components/history/__init__.py

```diff
@@ -124,6 +124,7 @@ class LazyState(history_models.LazyState):
         vol.Required("start_time"): str,
         vol.Optional("end_time"): str,
         vol.Optional("statistic_ids"): [str],
+        vol.Optional("period"): vol.Any("hour", "5minute"),
```

Comment:
Why optional? What is the default ?

Reply:
I think that it's better to be explicit here.

Reply:
OK, changed to a required parameter